db/migrate/20210401144737_set_first_and_second_course_years_for_core_induction_programme.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class SetFirstAndSecondCourseYearsForCoreInductionProgramme < ActiveRecord::Migration[6.1]
+  def up
+    CourseYear.all.each do |course_year|
+      cip = CoreInductionProgramme.find(course_year[:core_induction_programme_id])
+      if course_year.is_year_one?
+        cip.update!(course_year_one_id: course_year[:id])
+      else cip.update!(course_year_two_id: course_year[:id])
+      end
```

Comment:
That's incomplete - it only ever updates one year. What if the programme you are migrating has two course years on it?

Reply:
Line 5 had CourseYears.all.each but have switched this to go through each core induction programme, to follow the down method.